source/_patterns/01-atoms/alert/_alert.twig

```diff
@@ -0,0 +1,52 @@
+{#
+  Alert!
+
+  Documenting Alerts from Bootstrap.
+
+  type: String: status | warning | error: The type of error to display.
+  alert_degree: String (optional): The degree of error. Set via Type for Accessibility.
+  alert_title: String: The heading value of the Alert.
+  alert_primary: String: The paragraph value of the Alert.
+  alert_secondary: String (optional): The value of the footer.
+
+  see https://getbootstrap.com/docs/4.0/components/alerts/
+#}
+
+{% set status_classes = {
+  'status': 'success',
+  'error': 'danger',
+  'warning': 'warning',
+  'info': 'info',
+  }
+%}
+
+{% set alert_classes = [
+  'mb-0',
+  'alert',
+  'alert-' ~ status_classes[type],
+  'alert-dismissible',
+  'fade',
+  'show',
+] | sort | join(' ') | trim %}
+
+{% if type == 'error' %}
+  {% set alert_degree = 'role=alert' %}
```

Comment:
{% set alert_degree = ( type == 'error' ? 'role=alert' : 'aria-live=polite) %}` or whatever the cleanest shorthand is in PHP/Twig.